src/tests/ftest/deployment/disk_failure.yaml

```
@@ -0,0 +1,60 @@
+hosts:
+  test_servers: 4
+  test_clients: 1
+timeout: 420
+daos_server:
+  pattern_timeout: 60
+server_config:
+  name: daos_server
+  engines_per_host: 2
+  servers:
+    0:
+      pinned_numa_node: 0
+      nr_xs_helpers: 1
+      fabric_iface: ib0
+      fabric_iface_port: 31317
+      log_file: daos_server0.log
+      bdev_class: nvme
+      bdev_list: ["0000:00:00.0"]
+      scm_class: dcpm
+      scm_list: ["/dev/pmem0"]
+      scm_mount: /mnt/daos0
```

This won't work when merged with master.  The updated format is:
```suggestion
      storage:
        0:
          class: dcpm
          scm_list: ["/dev/pmem0"]
          scm_mount: /mnt/daos0
        1:
          class: nvme
          bdev_list: ["aaaa:aa:aa.a"]
```

---

Done.